### ISSUE TITLE
New language: Urdu

### ISSUE THREAD

**Issue by TheBluuDot** [squidfunk/mkdocs-material#4163]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### Language availability

- [X] I've checked the [list of available languages](https://bit.ly/33vFDD0)

### Translation

```jinja
{% macro t(key) %}{{ {
  "language": "ur",
  "direction": "rtl",
  "clipboard.copy": "کلِپ بورڈ میں نقل کریں",
  "clipboard.copied": "کلِپ بورڈ میں نقل کر دیا گیا",
  "edit.link.title": "اس صفحے میں ترمیم کریں",
  "footer.previous": "پچھلا",
  "footer.next": "اگلا",
  "footer.title": "ذیلی تحریر",
  "header.title": "سر تحریر",
  "meta.comments": "تبصرے",
  "meta.source": "ذریعہ",
  "nav.title": "رہنمائی",
  "search.placeholder": "تلاش کریں",
  "search.share": "اشتراک کریں",
  "search.reset": "صاف کریں",
  "search.result.initializer": "تلاش کا آغاز ہو رہا ہے",
  "search.result.placeholder": "تلاش شروع کرنے کے لئے ٹائپ کریں",
  "search.result.none": "کوئی ملتی جلتی دستاویزات نہیں",
  "search.result.one": "۱ ملتی جلتی دستاویز",
  "search.result.other": "# ملتی جلتی دستاویزات",
  "search.result.more.one": "اِس صفحے پر مزید ۱",
  "search.result.more.other": "اِس صفحے پر مزید #",
  "search.result.term.missing": "گمشدہ",
  "search.title": "تلاش",
  "select.language.title": "زبان کا انتخاب کریں",
  "select.version.title": "ورژن کا انتخاب کریں",
  "skip.link.title": "براہِ راست مواد پر جائیں",
  "source.link.title": "ریپازٹری پر جائیں",
  "source.file.date.updated": "آخری بار تجدید",
  "source.file.date.created": "تخلیق",
  "tabs.title": "ٹیبز",
  "toc.title": "فہرست",
  "top.title": "واپس اوپر جائیں"
}[key] }}{% endmacro %}
```

**Comment by squidfunk** [comment]
Thanks! Added in 53482b61b.

**Comment by squidfunk** [comment]
Released as part of 8.4.0.
